ci/ray_ci/docker_container.py

```diff
@@ -47,11 +48,22 @@ def __init__(
             ],
         )
 
-    def _get_image_version_tags(self) -> List[str]:
+    def _get_image_version_tags(self, external: bool) -> List[str]:
+        """
+        Get version tags.
+
+        Args:
+            external: If True, return the external image tags. If False, return the
+                internal image tags.
+        """
         branch = os.environ.get("BUILDKITE_BRANCH")
         sha_tag = os.environ["BUILDKITE_COMMIT"][:6]
         pr = os.environ.get("BUILDKITE_PULL_REQUEST", "false")
+        formatted_date = datetime.now().strftime("%y%m%d")
+
         if branch == "master":
+            if external:
+                return [f"nightly.{formatted_date}.{sha_tag}", "nightly"]
```

Comment:
can we remove the `"nightly"` on the non-external case?

and also can we condition returning `nightly` only on `RAYCI_SCHEDULE=nightly` ?

Reply:
done